contracts/core/tokens/rebalancing-v2/RebalancingSetState.sol

```diff
@@ -201,6 +201,18 @@ contract RebalancingSetState {
 
     /* ============ Getter Functions ============ */
 
+    /*
+     * Retrieves the current expected fee from the fee calculator
+     * Value is returned as a scale decimal figure.
+     */
+    function rebalanceFee()
```

Comment:
Hmmm won't there be times where this doesn't return a number? For a constant fee it works but say the fee calculation requires knowing what the nextSet will be?

Reply:
In other implementations, this function could revert or return an anticipated value